src/cse-machine/utils.ts

```diff
@@ -503,56 +503,107 @@ export const checkStackOverFlow = (context: Context, control: Control) => {
   }
 }
 
-/**
- * Checks whether a function body returns in every possible branch.
- * Returns true if every branch has a return statement, else returns false.
- * @param body The function body to be checked
- */
```

Comment:
Why was this documentation removed?

Reply:
Sorry for the error, I've mistakenly implemented a wrong version of `hasReturnStatement`. I've just fixed the function and put the documentation back.